src/components/Recipe.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {Link} from "react-router-dom"
+import apiClient from "../common/apiClient";
+import {Card, ListItem, SmallButton} from "../common/styled";
+
+function Recipe(props) {
+    function deleteRecipe() {
+        if (window.confirm("Are you sure?")) {
+            async function remove() {
+                await apiClient.remove(props.value.id)
+                alert("Recipe deleted!")
+                props.refreshList()
+            }
+
+            remove()
+        }
+
+    }
+
+    return (
+        <Card data-testid={"recipe"+props.value.id}>
```

Comment:
We try to use string interpolation with template literals this way everywhere
with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals